api/v1alpha1/selfnoderemediationconfig_types.go

```diff
@@ -127,6 +128,11 @@ type SelfNodeRemediationConfigSpec struct {
 	// CustomDsTolerations allows to add custom tolerations snr agents that are running on the ds in order to support remediation for different types of nodes.
 	// +optional
 	CustomDsTolerations []v1.Toleration `json:"customDsTolerations,omitempty"`
+
+	// +kubebuilder:default:=1
+	// +kubebuilder:validation:Minimum=0
+	// Minimum number of peer workers/control nodes to attempt to contact before deciding if node is unhealthy or not
```

Comment:
please add what happens when this is set 0
nit: please move comment above the kubebuilder markers

Reply:
also: does it make sense allow a higher value than 1? If so, what happens when we have less workers available? :thinking:

Reply:
In case higher value does not make sense, I suggest switching to a boolean value

Reply:
> also: does it make sense allow a higher value than 1? If so, what happens when we have less workers available? 🤔

We wanted to make this configurable such that for different configs that we have this being used in - different amounts of nodes and differing use cases, we gave flexibility of choice.

I do believe that in general it's going to be either zero or one, but didn't want to make that change when one of our configurations drive it.

Reply:
> please add what happens when this is set 0 nit: please move comment above the kubebuilder markers

I added more details and moved the comments

Reply:
@slintes checking to see if this can be resolved based on the latest changes.  Thanks!

Reply:
You need to run make bundle for updating CRD files with the updated comment. Other than that lgtm 👍🏼